pkg/aggregator/internal/limiter/limiter.go

```diff
@@ -27,10 +29,17 @@ type Limiter struct {
 	keyTagName        string
 	telemetryTagNames []string
 	limit             int
+	global            int // global limit
+	current           int // sum(usage[*].current)
 	usage             map[string]*entry
+
+	// epoch, maxAge and lastSeen ensure eventual removal of entries that created an entry, but were
+	// never able to create contexts due to the global limit.
+	epoch  int
+	maxAge int
```

Comment:
```suggestion
	maxCycleAge int
```

Something to associate this and above (currently named) `epoch` as having the same units and being related to one another